src/utils.test.js

```diff
@@ -60,4 +62,60 @@ describe('utils', () => {
       expect(dataHubParams.hash).toBe('#hash1#hash2');
     });
   });
+
+  describe('setInterval custom hook', () => {
+    it('should set an interval and update an outside value', () => {
+      jest.useFakeTimers();
+      let counter = 0;
+      renderHook(() =>
+        useInterval({
+          runOnStart: false,
+          delay: 100,
+          callback: () => {
+            counter++;
+          },
+        }),
+      );
+
+      jest.runOnlyPendingTimers();
+      expect(counter).toBe(1);
+    });
```

Comment:
nice! maybe can add the comments to separate sections like `Arrange, Act, Assert`

Reply:
I will try, this test is very still very alien to me.

Reply:
@fcoronelmakingsense solved!